Jenkinsfile

```diff
@@ -75,22 +75,30 @@ pipeline {
                     Globals.dockerCacheArg = (env.param_enable_docker_cache.toBoolean()) ? '' : '--no-cache'
                     println("Docker cache args: " + Globals.dockerCacheArg)
 
-                    Globals.dockerLabels = "--label git_revision=${Globals.rootCommit} --label build_id=${env.BUILD_NUMBER} --label build_user=${env.BUILD_USER} --label build_date=${buildDate}"
+                    wrap([$class: 'BuildUser']) {
+                        if (binding.hasVariable('BUILD_USER ')) {
+                            BUILD_USER = "${BUILD_USER}"
+                        } else {
+                            BUILD_USER = null
+                        }
+                    }
+
```

Comment:
Can we make it a bit smoother?
smth like this for example:
BUILD_USER = (binding.hasVariable('BUILD_USER ') ? "${BUILD_USER}" : 'null'

Reply:
Yeah, sure